tests/vesting/VestingCreator.js

```diff
@@ -893,127 +861,6 @@ contract("VestingCreator", (accounts) => {
         });
     });
```

Comment:
this was removed because the tests already covered in the "vestingRegistry" and since we are removing vestingRegistry3 , it does make sense to also removed the redundant test